indico/modules/categories/models/categories.py

```diff
@@ -287,6 +287,9 @@ def get_chain_titles(self):
         """Retrieve the category titles from the root to the category"""
         return [c.title for c in self.chain_query]
 
+    def get_non_inheriting_objects(self):
```

Comment:
Have a look at `IndicoProtectionField.render_protection_message` method. Either we override the method here or we put a proper `if` statement in the `render_protection_message`